common/djangoapps/third_party_auth/management/commands/tests/test_saml.py

```diff
@@ -170,7 +169,7 @@ def test_fetch_saml_metadata(self):
         # Create enabled configurations
         self.__create_saml_configurations__()
 
-        expected = "\nDone.\n1 provider(s) found in database.\n0 skipped and 1 attempted.\n1 updated and 0 failed.\n"
+        expected = "\nDone.\n2 provider(s) found in database.\n1 skipped and 1 attempted.\n1 updated and 0 failed.\n"
```

Comment:
Can you explain why removal of the toggle results in an additional provider?

Reply:
@ktyagiapphelix2u: I'm still waiting on a response here. I also don't understand what "1 updated" means. Nothing actually gets updated, right?

Reply:
@ktyagiapphelix2u: I continue to not get a response for this comment, and it affects much of the review, so I don't plan on proceeding until I hear back.
- I still want to know what "skipped" and "attempted" are, and whether or not these are leftover from when the command used to do something, rather than just report out? We probably shouldn't mess with anything in this PR, but maybe we should do a simple clean-up, if this is outdated?
- Should we drop the call to `self._create_saml_configurations()` if there is already enough test data for this test? Same with some other tests? When do we actually need to call this? I still find that the test is more confusing than it needs to be, especially from a review perspective, wondering why the expectation needs to change.